### PR TITLE
Change default keypair keypath for Docker-based local tests.

### PR DESCRIPTION
**Pull request by rohinb2**
As explained in `tests/README.md`, users generally have a sky-key set up and we can use that for communication with local Docker clusters.

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ do TDD and add edge cases one-off as needed. We also use imports within test mod
 test cases in multiple suites where appropriate (just importing the test case function is enough for PyTest to
 do this, so ignore warnings that the imports are unused).
 
+Runhouse uses Sky (which uses ssh) to communicate with the clusters it runs on (e.g. AWS EC2). Sky generates
```
Comment (dongreenberg) [inline tests/README.md:22]:
```suggestion
Runhouse uses SkyPilot (which uses ssh) to communicate with the clusters it runs on (e.g. AWS EC2). Sky generates
```

File: tests/README.md [hunk 2]
```diff
@@ -19,6 +19,10 @@ do TDD and add edge cases one-off as needed. We also use imports within test mod
...
+a keypair for you locally the first time you use it to communicate with a cluster, and for local container tests
+we take this key from `~/.ssh/sky-key` and copy it to the Docker containers such that Sky can communicate with them.
```
Comment (dongreenberg) [inline tests/README.md:24]:
```suggestion
we take this key from `~/.ssh/sky-key` and copy it to the Docker containers such that Sky can communicate with them. You can override the choice of keypair by adding the following line to your `~/.rh/config.yaml`: `default_keypair: <path to your private key>`.
```

**Review by dongreenberg — APPROVED** [review]
